src/prompts/list.ts

```diff
@@ -10,7 +10,12 @@ export default class ListPrompt extends Prompt {
 
 	public render() {
 		const choices = this._question.choices.reduce((result, choice) => {
-			result[choice.name] = choice.value;
+			// choices can be string array or array of { name, value }
+			if (choice.name) {
```

Comment:
I have a better idea. Remove these changes and put back what we had before. Now, instead of letting the `ListPrompt` extend from `Prompt`, let it extend from `CheckboxPrompt`. The constructor of the `CheckboxPrompt` already convert these objects so we get that for free.

Later on we can then easily improve this (see #50).